Structure_Engine/Query/ReinforcementLayout.cs

```diff
@@ -203,8 +221,11 @@ public static List<ICurve> ReinforcementLayout(Bar bar)
         [Input("length", "Length of the host Bar used to generate the lines", typeof(Length))]
         [Input("transformation", "Transformation needed to move the lines from the position of the host element.")]
         [Output("points", "The centrelines of the LongitudinalReinforcement.")]
-        public static List<Line> ReinforcementLayout(this LongitudinalReinforcement reinforcement, double cover, List<ICurve> outerProfileEdges, List<ICurve> innerProfileEdges, double length, TransformMatrix transformation)
+        public static List<Line> ReinforcementLayout(this LongitudinalReinforcement reinforcement, double cover, List<ICurve> outerProfileEdges, List<ICurve> innerProfileEdges, double length, TransformMatrix transformation = null)
```

Comment:
Not sure I agree with making this argument optional, especially as it is not handled. Would revert this change, requiring a transformation matrix to be provided.